### ISSUE TITLE
pass-enoaudit flag passes integration even without registry returning ENOAUDIT

### ISSUE THREAD

**Issue by sumanapotturu** [IBM/audit-ci#107]
Test: 
```
pass-enoaudit: true
critical: true
```

Expected Behavior:
If registry does not return ENOAUDIT, the expected behavior is the integration fails with error
```
    "vulnerabilities": {
      "info": 0,
      "low": 18,
      "moderate": 47,
      "high": 402,
      "critical": 1
    }
Failed security audit due to critical vulnerabilities.
Exiting...

```

Current Behavior:
Registry is not returning ENOAUDIT  and audit-ci passes the integration even with critical vulnerabilities
```
    "vulnerabilities": {
      "info": 0,
      "low": 18,
      "moderate": 47,
      "high": 402,
      "critical": 1
    }
ACTION RECOMMENDED: An audit could not performed due to 5 audits that resulted in ENOAUDIT. Perform an audit manually and verify that no significant vulnerabilities exist before merging.
Passed npm security audit.
```

**Comment by quinnturner** [comment]
Confirmed reproduction, will work on a fix immediately.

**Comment by sumanapotturu** [comment]
Fix is to change line 27 in lib/audit.js to 
if (config['pass-enoaudit'] && message.includes(RETRY_ERROR_MSG[pm])) {

**Comment by sumanapotturu** [comment]
I worked on the fix, unable to push the code. Permission denied

**Comment by quinnturner** [comment]
If you'd like to submit your fix, create a fork of the project, push to your fork, then create a PR :)

**Comment by quinnturner** [comment]
Closed with #108
